frontend/src/components/DropdownMenu.js

```diff
@@ -83,7 +83,7 @@ export default function DropdownMenu({
     );
   }
   return (
-    <div role="menu" ref={forwardedRef} tabIndex="-1" aria-label={menuName} className={classNames} onBlur={onBlur} onKeyDown={onKeyDown}>
+    <div role="menu" ref={forwardedRef} tabIndex="0" aria-label={menuName} className={classNames} onBlur={onBlur} onKeyDown={onKeyDown}>
```

Comment:
When I tab through the page I stay on the `filters` dropdown for two presses of my tab key. The first time when I press enter nothing happens. The next element that is focused on tab opens the dropdown menu when I press enter. Took a screenshot of the first element in focus
![Screenshot from 2021-11-29 17-22-38](https://user-images.githubusercontent.com/6631790/143958359-125484a7-8f02-4822-8cc6-b65ee1a8eeb7.png)
and the second
![Screenshot from 2021-11-29 17-22-43](https://user-images.githubusercontent.com/6631790/143958395-1f26508b-7afc-40ee-878c-207d8a9cd4a1.png)

Reply:
Really scratching my head as to why I changed this